thirdeye/thirdeye-frontend/tests/integration/helpers/calculate-direction-test.js

```diff
@@ -0,0 +1,20 @@
+
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('calculate-direction', 'helper:calculate-direction', {
+  integration: true
+});
+
+test('when input is positive, calculate direction should return "up"', function(assert) {
+  this.set('inputValue', '1234');
+  this.render(hbs`{{calculate-direction inputValue}}`);
+  assert.equal(this.$().text().trim(), 'up');
```

Comment:
nit: let's pass a meaningful message as the 3rd argument, we'll thank ourselves later for that